src/unk_02055418.c

```diff
@@ -24,7 +24,7 @@
 void FieldSys_StartBugContestTimer(FieldSystem* fsys) {
     RTCDate date;
     RTCTime time;
-    SYSINFO_RTC* sysinfo = Save_SysInfo_RTC_Get(fsys->savedata);
```

Comment:
Pointer notation in this file

Reply:
out of scope - will be addressed in another PR, when I get to the file

Reply:
it's literally a cntrl + f and replace smh